types/query-helpers.d.ts

```diff
@@ -69,6 +69,6 @@ export type BuiltQueryMethods<Arguments extends any[]> = [
 
 export function buildQueries<Arguments extends any[]>(
   queryAllBy: GetAllBy<Arguments>,
-  getMultipleError: GetErrorFunction,
-  getMissingError: GetErrorFunction,
+  getMultipleError: GetErrorFunction<Arguments>,
```

Comment:
Is there a way to add a [test]((https://github.com/testing-library/dom-testing-library/blob/main/types/__tests__/type-tests.ts) for this?

Reply:
Added here: https://github.com/testing-library/dom-testing-library/pull/1041/commits/ece7eb7c58bec2f4e6a25803758431c80450cefc